blockonomics-woocommerce.php

```diff
@@ -97,6 +97,7 @@ function add_page()
     {
         include_once plugin_dir_path(__FILE__) . 'php' . DIRECTORY_SEPARATOR . 'Blockonomics.php';
         $blockonomics = new Blockonomics;
+        update_option('blockonomics_active_cryptos',$blockonomics->getActiveCurrencies());
```

Comment:
Adding a new option *blockonomics_active_cryptos* is a bad design as it is duplicating data. This information can be easily fetched via *getActiveCurrencies()* which is a very cheap function call. Remove this option.

Reply:
Removed in b786b57 and b6945c4